rpc/clients/typescript/src/ws_client.ts

```diff
@@ -336,6 +336,24 @@ export class WSClient {
         };
         return getOrdersResponse;
     }
+    /**
+     * Get page of 0x signed orders stored on the Mesh node at the specified snapshot
+     * @param page Page index at which to retrieve orders
+     * @param perPage number of signedOrders to fetch per paginated request
+     * @param snapshotID The DB snapshot at which to fetch orders. If omitted, a new snapshot is created
+     * @returns the snapshotID, snapshotTimestamp and all orders, their hashes and fillableTakerAssetAmounts
+     */
+    public async getOrdersForPageAsync(page: number, perPage: number = 200, snapshotID?: string): Promise<GetOrdersResponse> {
```

Comment:
@fabioberger can we use this new `getOrdersForPageAsync` in the implementation of `getOrdersAsync` to make the code DRYer?

Reply:
Great idea, done ✅